docs/source/setup.md

```diff
@@ -202,15 +202,15 @@ npm install -g bower
 
 ### Step 4: Start the development environment
 
-* To run backend development server at [backend]
+* To run backend development server at [backend]:
```

Comment:
Please specify the port number as `127.0.0.1:8000` here as done in above command.